_includes/c11-cpp11-and-beyond-and-toolchains.md

```diff
@@ -8,7 +8,7 @@ If your project requires tools compatible with C11, C++11, or a more recent lang
 * [Trusty](/user/reference/trusty) ships with GCC 4.8.2
 * [Xenial](/user/reference/xenial) ships with GCC 5.4.0
 
-Note that [GCC support for ISO C11 reached a similar level of completeness as ISO C99 in 4.9](https://gcc.gnu.org/wiki/C11Status) and that C++11 is feature-complete in 4.8.1, but [support for `<regex>` does not exist until 4.9](https://gcc.gnu.org/bugzilla/show_bug.cgi?id=53631).
+Note that [GCC support for ISO C11 reached a similar level of completeness as ISO C99 in 4.9](https://gcc.gnu.org/wiki/C11Status) and that C++11 is feature-complete in 5.1 (the compiler was feature-complete in 4.8.1 but the standard library didn't support all C++11 features until [later](https://gcc.gnu.org/gcc-5/changes.html#libstdcxx), in particular [support for `<regex>` does not exist until 4.9](https://gcc.gnu.org/bugzilla/show_bug.cgi?id=53631)).
```

Comment:
Looks great thanks.

Does this apply to C also? or just C++? This file is reused in both

Reply:
The first part of the sentence about C is accurate, so I didn't touch that. With my changes the file continues to be applicable to both C11 and C++11, but now the C++11 info is accurate :smiley:

Reply:
Would it help if I said "(the C++ compiler was feature-complete in 4.8.1 ..." so it's clear the parenthetical remark only applies to C++?

Reply:
Or maybe "the C++ language support was feature-complete in 4.8.1 ..." to distinguish language from library.

Reply:
That last sounds best to me! Thanks!